### PR TITLE
Added handlebars support

### PR DESCRIPTION
**Pull request by pajasevi**
Hi,

I wanted to add Handlebars support - includes templates and partials.

Here's initial code, tested (by hand, tests do not cover templates).

Feel free to review and comment.

Thanks for your work! :heart_eyes:

### CODE REVIEW

File: lib/tasks/templates.js [hunk 1]
```diff
@@ -8,10 +8,14 @@ module.exports = function(gulp, config) {
 
 		var jade = require('jade')
 		var gulpJade = require('gulp-jade')
+		var handlebars = require('handlebars')
```
Comment (enzy) [inline lib/tasks/templates.js]: This module is not used in following code

File: lib/tasks/templates.js [hunk 2]
```diff
@@ -8,10 +8,14 @@ module.exports = function(gulp, config) {
...
+		var gulpHandlebars = require('gulp-static-handlebars')
```
Comment (enzy) [inline lib/tasks/templates.js]: Which reasons led you to use this library instead of https://github.com/shannonmoeller/gulp-hb (for ex.)?

**Comment by pajasevi** [comment]
Yeah, you're right, I will remove that, since it's not used.
I used this library simply because I didn't find the other - which seems better. Do you want me to rewrite that and use gulp-hp instead?

**Comment by enzy** [comment]
No need. I'm thinking about working on your code in a way I pointed out in #47 (but no ETA for now).

**Comment by pajasevi** [comment]
Ok, I removed that unused module at least. Feel free to contact me, if you need anything.:calling:

**Pull request closed without merging**